index.html

```diff
@@ -59,7 +59,13 @@
       </div>
       <div class="clear"></div>
     </nav>
+    <!-- important links -->
+    <div id="links"> 
+      <a href="https://app.datadoghq.com/dashboard/nyc-69g-cpr/patricks-dashboard?from_ts=1671546591056&to_ts=1671560991056&live=true" target="_blank" class="link-text">
+        Lens Datadog
```

Comment:
@sarahmangels This text color is showing up as the default blue instead of the `#333` used everywhere else- can we update that? Additionally, let's use a `text-decoration: none` to remove the underline, I think it aligns more closely with the existing styles.

Reply:
ok steve jobs